set1/Q1.cpp.txt

```diff
@@ -0,0 +1,27 @@
+// Question link : https://practice.geeksforgeeks.org/problems/k-largest-elements/0
+#include <iostream>
+#include<algorithm>
+#include<vector>
+using namespace std;
+vector<int> arr;
+
+int main()
+{
+	int t, n, k;
+	//freopen("Testcase.txt", "r", stdin);
+	//freopen("output.txt", "w", stdout);
+	cin >> t;
+	while (t-- > 0) {
+		cin >> n >> k;
+		for (int i = 0; i < n; i++) {
+			cin >> arr[i];
+		}
+		make_heap(arr.begin(), arr.end());
+		for (int i = 0; i < k; i++) {
+			cout << arr[0] << " ";
+			pop_heap(arr.begin(), arr.end());
+		}
+	}
+	return 0;
+}
+
```

Comment:
Also it would be better if you explain a little bit and mention the complexity of the code in comments.